kotlin-jvm/src/main/kotlin/playground/FuelHttp.kt

```diff
@@ -0,0 +1,19 @@
+package playground
+
+import com.github.kittinunf.fuel.core.FuelManager
+import java.net.URL
+
+fun main() {
+    println()
+    println("# kittinunf /fuel - The easiest HTTP networking library for Kotlin/Android")
+
+    val url = "http://httpbin.org"
+    val httpClient = FuelManager.instance
+
+    val (_, response, _) = httpClient
+        .get(url, listOf("hello" to "world"))
+        .responseString()
+
+    response.statusCode shouldBe 200
+    response.url shouldBe URL("http://httpbin.org?hello=world")
```

Comment:
It would be nice if you show how the result can be deserialized with kotlinx-serialization
https://fuel.gitbook.io/documentation/deserialization/fuel-kotlinx-serialization

FYI the response should be of type `HttpBin` which is already defined
https://github.com/LouisCAD/kotlin-libraries-playground/blob/017e0a99561d9b7494ac8e4cddc7626a0149ca39/kotlin-jvm/src/main/kotlin/playground/KotlinxSerialization.kt#L60-L66